spec.html

```diff
@@ -8620,9 +8642,13 @@ <h1>ModuleNamespaceCreate ( _module_, _exports_ )</h1>
 
     <emu-clause id="sec-immutable-prototype-exotic-objects">
       <h1>Immutable Prototype Exotic Objects</h1>
-      <p>An <dfn>immutable prototype exotic object</dfn> is an exotic object that has a [[Prototype]] internal slot that will not change once it is initialized.</p>
+      <p>An immutable prototype exotic object is an exotic object that has a [[Prototype]] internal slot that will not change once it is initialized.</p>
+
+      <p>An object is said to be an <dfn id="immutable-prototype-exotic-object">immutable prototype exotic object</dfn> if its [[SetPrototypeOf]] internal method uses the following implementation.</p>
 
-      <p>Immutable prototype exotic objects have the same internal slots as ordinary objects. They are exotic only in the following internal methods. All other internal methods of immutable prototype exotic objects that are not explicitly defined below are instead defined as in <a href="#sec-ordinary-object-internal-methods-and-internal-slots">ordinary objects.</a></p>
+      <emu-note>
+        <p>Unlike other exotic objects, there is not a dedicated creation abstract operation provided for immutable prototype exotic objects. This is because they are only used by %ObjectPrototype% and by host environments, and in host environments, the relevant objects are potentially exotic in other ways and thus need their own dedicated creation operation.</p>
```

Comment:
I wondered whether we should add an ImmutablePrototypeObjCreate, but I'm ok with not having it for now since we would host object to define there on xxxCreate operations that incorporate this.  Perhaps the sort of thing that should be explained in a FAQ on defining "host objects"
```suggestion
        <p>Unlike other exotic objects, there is not a dedicated creation abstract operation provided for immutable prototype exotic objects. This is because they are only used by %ObjectPrototype% and by host environments, and in host environments, the relevant objects are potentially exotic in other ways and thus need their own dedicated creation operation.</p>
```